### PR TITLE
NXP-22752: allow editing both users and groups

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js [hunk 1]
```diff
@@ -21,18 +21,33 @@ class UserEditor extends Select2Editor {
...
   saveValue(val, ctrlDown) {
     // create directory entries again on save
     var value = val[0][0];
 
     if (value) {
       value = value.split(',').map(function (id) {
+        let type = 'user'; // XXX: can't guess type if not prefixed
+
+        if (id.startsWith('user:') || id.startsWith('group:')) {
+          //if (this.widgetProperties.prefixed) {
```
Comment (Gabez0r) [inline nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js]: do we need to keep this commented code?

File: nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js [hunk 2]
```diff
@@ -21,18 +21,33 @@ class UserEditor extends Select2Editor {
...
+          let parts = id.split(':');
+          type = parts[0]; id = parts[1];
+          //}
```
Comment (Gabez0r) [inline nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js]: same here

**Comment by Gabez0r** [comment]
👍

File: nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js (hunk 1 above)
Comment (nelsonsilva) [reply nuxeo-platform-spreadsheet-web/src/main/js/app/ui/editors/user.js]: no we don't 👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**